src/index.html

```diff
@@ -11,6 +11,71 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form 
```

Comment:
No space between parent and child element
```suggestion
    <form 
```